apps/dashboard/src/components/inbox-button.tsx

```diff
@@ -0,0 +1,27 @@
+import { Popover, PopoverContent, PopoverTrigger } from '@/components/primitives/popover';
+import { APP_ID } from '@/config';
+import { Bell, Inbox, InboxContent } from '@novu/react';
+import { PopoverPortal } from '@radix-ui/react-popover';
+
+export const InboxButton = () => {
+  //   const { user } = useUser();
```

Comment:
Not sure how we will handle dev env vars yet so clerk instance wasn't there.

Reply:
We will use .env as Vite says. Does this help?